examples/scripts/smart-coffee-machine.js

```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
- * Copyright (c) 2018 - 2020 Contributors to the Eclipse Foundation
+ * Copyright (c) 2018 - 2022 Contributors to the Eclipse Foundation
```

Comment:
I think we end up choosing just one year, not the year range right?

Reply:
Good point, I fixed the files I touched in this PR. see https://github.com/eclipse/thingweb.node-wot/pull/669/commits/cc227af18d6188ae07dca1fdede2b600a5f17ced
I will do another run with the script to check copyright years for all files also.